extensions/amp-analytics/0.1/instrumentation.js

```diff
@@ -89,24 +89,20 @@ export class InstrumentationService {
    * @return {!./analytics-root.AnalyticsRoot}
    */
   findRoot_(context) {
-    // FIE
-    const frame = getParentWindowFrameElement(context, this.ampdoc.win);
-    if (frame) {
-      const embed = getFriendlyIframeEmbedOptional(frame);
+    // TODO(#22733): cleanup when ampdoc-fie is launched.
```

Comment:
could you be more clear about this TODO. My understanding that before ampdoc-fie launch the ampdoc returned by L93 will still be the same, so we need `embed` to be the holder. After the launch, embed can be removed from this method?